src/lib/secretGen.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const env = require('./env');
+
+/**
+ * Generate secret
+ * @param {object|string} options
+ * @param {string} options.app Application name
+ * @param {string} options.region Application region
+ * @param {string} options.env Application environment
+ * @param {string} options.name Variable name
+ * @return {string}
+ */
+module.exports = function(options = { app: '', region: '', env: '', name: '' }) {
+    if(typeof options === 'string')
```

Comment:
So both cases return a string. The object notation is if you want to specify each yourself which is not the usual case (yadda uses this method). Inside of the client-side use of this you would pass a string through and it will pull from the environment what it needs.